src/3d/qgs3dmapsettings.h

```diff
@@ -325,12 +326,24 @@ class _3D_EXPORT Qgs3DMapSettings : public QObject, public QgsTemporalRangeObjec
      */
     QList<QgsPointLightSettings> pointLights() const { return mPointLights; }
 
+    /**
+     * Returns list of directional lights defined in the scene
+     * \since QGIS 3.16
+     */
+    QList<QgsDirectionalLightSettings> directionalLights() const { return mDirectionalLights; }
+
     /**
      * Sets list of point lights defined in the scene
      * \since QGIS 3.6
      */
     void setPointLights( const QList<QgsPointLightSettings> &pointLights );
 
+    /**
+     * Sets list of directional lights defined in the scene
+     * \since QGIS 3.16
+     */
+    void setDirectionalLights( const QList<QgsDirectionalLightSettings> &pointLights );
```

Comment:
```suggestion
    void setDirectionalLights( const QList<QgsDirectionalLightSettings> &directionalLights );
```

For consistency?

Reply:
My bad, I just copied the signature and forgot to edit it